src/app/_components/CardSpecialist/CardWrapper.js

```diff
@@ -0,0 +1,41 @@
+'use client';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useMediaQuery } from '@mui/material';
+import { useRouter, useSearchParams } from 'next/navigation';
+import { useHintContext } from '@components';
+import { cn } from '@utils/cn';
+import { screens } from '@/app/styles/tailwind/ui';
+
+export function CardWrapper({ children, className, id }) {
+  const router = useRouter();
+  const matches = useMediaQuery(`(max-width: ${screens.md})`);
+  const params = useSearchParams();
+  const { toggle } = useHintContext();
+
+  const isExtendedCardOpened = !!params.get('id');
+  const cursorStyle = isExtendedCardOpened ? 'cursor-auto' : 'cursor-pointer';
```

Comment:
You can pass it as an object to cn, cn({'cursor-pointer': !isExtendedCardOpened, 'cursor-auto': isExtendedCardOpened})

Reply:
Done